### PR TITLE
adding OHMA decoding

### PR DESCRIPTION
**Pull request by makrsmark**
used on 737 MAX

### CODE REVIEW

File: lib/plugins/Label_H1_OHMA.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    decodeResult.decoded = true;
+    decodeResult.decoder.decodeLevel = 'full';
+    decodeResult.raw.ohma = jsonText;
+    decodeResult.formatted.items.push({
+      type: 'ohma',
+      code: 'OHMA' ,
+      label: 'OHMA Downlink',
+      value: JSON.stringify(ohmaMsg, null, 2)
+    });
```
Comment (makrsmark) [inline lib/plugins/Label_H1_OHMA.ts:43]: we can probably decode further... i'd like to look at the json first

**Comment by kevinelliott** [comment]
Sorry for delay! I meant to review this earlier today.

File: lib/plugins/Label_H1_OHMA.test.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    preambles: ['OHMA', '/RTNBOCR.OHMA', '#T1B/RTNBOCR.OHMA'],
+  });
+});
+
+test('decodes Label H1 Preamble OHMA valid', () => {
+  const decoder = new MessageDecoder();
+  const decoderPlugin = new Label_H1_OHMA(decoder);
+
+  // https://app.airframes.io/messages/3126721669
+  const text = 'OHMAeJy1Ul2PojAU/SubPiuhlA/ljQHMkAElgDuTXTakqx2nCaAp1WQz8b/vFXTGEcw+LX26veece88p7+jARMO3NbKRpqhohCrWNHTDoH7P0arkrJbBOkd2jhaPkZOjUX6BeFSytqGpmj5WJ2NVyzC2ydTWiWJN9B8teE0lBRSIUS52Ja1ZA+VPqCXl5Xxf/WaiVZkTyzD0bsB2zcr2si1fS755kxfamu2okHvBHNDbCukCuMU+PSzSFk+F4Ada9vqLxLvSuxq9dELDtK56IdukEqZkvOpbVG0VXIJFYnYW2QFCuqzXFm5Jm+YcG2xLKyYFX32CB3V70XXQPzv2EX8R+Vn3BrxuJK1XH2HW9KxYyRgWv5gDJCSx72CnOH8dR1/RLhXl9+4f6MF1xRhgJPuSNUzeI1nE+hY5L7ZtO4/RmFjRQwj+4MNqCJf4pNhp/reosggXbhjEke8Fzrxwk2WQ+kUSZ8U/wgNi5kcxhpy9G2PENBVtqt+kcSZoQ4QJViyTDBO8YDYb4GBDwQTfUOIIx4mfpsvEH1xMhzHmfc7gbkTR1R7HcZ9m4eL53m5YmeAhTjA/jbrDMhULT08P3p0jOv4FG0dfvQ==';
```
Comment (kevinelliott) [inline lib/plugins/Label_H1_OHMA.test.ts:22]: Oh sweet! base64 encoded and zlib compressed! Sweet juicy JSON inside. Nice job here.

**Review by kevinelliott — APPROVED** [review]
Nice find. LGTM!

**Pull request merged**